modules/nextflow/src/main/groovy/nextflow/container/PodmanBuilder.groovy

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.container
+import groovy.transform.CompileStatic
+/**
+ * Helper methods to handle podman containers
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ * @author tbugfinder <github@online.ms>
+ */
+@CompileStatic
+class PodmanBuilder extends ContainerBuilder<PodmanBuilder> {
+
+    private boolean sudo
+
+    private boolean remove = true
+
+    private boolean userEmulation
```

Comment:
Not needed.

Reply:
removed userEmulation with
commit 13b0fe2072ffb3202d3d0832ceebf4f991f7944a

Reply:
I'm sorry - I've added userEmulation and USER_AND_HOME_EMULATION again as I thought it's a useful shortcut. Please let me know if it's ok or should be finally removed.

Reply:
It was more an hack and with podman it should not be necessary. I would say to remove it.

Reply:
removed by 
commit 7dfa7c9169892505f45c3d42d13e83e2447f2ec9